web/src/components/ChatBotWidget.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { ReactElement, useEffect } from 'react'
-import { useTheme } from 'styled-components'
+import styled, { useTheme } from 'styled-components'
+
+const ChatContainer = styled.div`
+  > div {
+    right: calc(100% - 100vw + 33px);
```

Comment:
where does this 33px come from?

Reply:
It's a magic number consisting of the average 17px that a scroll bar usually has as width + the original 16px that the box had as padding.